src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -116,20 +112,33 @@ def run_offline_reintegration_test(self, num_pool, data=False):
             self.pool.display_pool_daos_space("Pool space: Beginning")
             pver_begin = self.get_pool_version()
             self.log.info("Pool Version at the beginning %s", pver_begin)
-            output = self.dmg_command.pool_exclude(self.pool.uuid,
-                                                   rank)
+            if server_boot is False:
+                output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                       rank)
+            else:
+                output = self.dmg_command.system_stop(ranks=rank)
+                time.sleep(10)
+                self.log.info(output)
+                self.is_rebuild_done(3)
+                self.assert_on_rebuild_failure()
+                time.sleep(35)
```

Comment:
Can you help me understand why do we have sleep here if self.is_rebuild_done(3) is checking for rebuild completion at intervals of 3.

Reply:
Removed.